### PR TITLE
link to 2i2c and MD conduct/community guidelines

### PR DESCRIPTION
**Pull request by jmunroe**
Clarify that ScienceCore:climaterisk should be consistent with both the 2i2c Code of Conduct and MetaDocencia Community Guidelines.

### CODE REVIEW

**Comment by NPalopoli** [comment]
Here's the link to MD [Community Guidelines](https://www.metadocencia.org/en/pdc/)

File: CONDUCT.md [hunk 1]
```diff
@@ -1,44 +1,8 @@
...
-## Enforcement
-
-Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
-
-Project maintainers who do not follow or enforce the Code of Conduct in good faith may face temporary or permanent repercussions as determined by other members of the project's leadership.
+As a joint-project between 2i2c and MetaDocencia, ScienceCore:climaterisk is subject to both of the following policies:
 
-## Attribution
+- [2i2c Code of Conduct](https://compass.2i2c.org/code-of-conduct)
+- [MetaDocencia Community Guidelines](https://www.metadocencia.org/en/pdc)
```
Comment (NPalopoli) [inline CONDUCT.md:6]: To encourage a friendly response to Spanish-speaking persons, we can link to our Spanish version instead: [Pautas de Convivencia de MetaDocencia](https://www.metadocencia.org/pdc/)

**Review by NPalopoli — CHANGES_REQUESTED** [review]
Since this is a short document now, I don't see the point of having a different CONDUCT.md file for each language. 

Unless 2i2c's CoC is also available in Spanish, I suggest we link to MD's Spanish version instead.

We can then have two separate sections, one in each language, in this same document.

File: CONDUCT.md (hunk 1 above)
Comment (jmunroe) [reply CONDUCT.md]: Makes sense to me. Changed.

**Review by NPalopoli — APPROVED** [review]

**Pull request merged**